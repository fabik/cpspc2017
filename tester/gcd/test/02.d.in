15
27 11 25 28 21 28 19 29 23 16 21 10 17 29 16
