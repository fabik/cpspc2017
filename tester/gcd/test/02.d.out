1
30
59
87
115
142
167
190
211
232
251
268
284
304
320
